src/components/CommentsList.jsx

```diff
@@ -29,25 +29,23 @@ function CommentsList (){
             )
     } else {
         return(
-            <div>
                 <div>
                     <p className="big-comments">Comments:
                     </p> 
                     {comments.map((comment) => {
                         return (
                         <>
```

Comment:
Because it's only returning one div, don't need the fragment `<></>`
Also there's a unique key problem, it would be fixed when the react fragment is removed